styles/variables-silicon.less

```diff
@@ -362,3 +362,10 @@
 // Tooltip
 // ---------------------------------------
 @agate-tooltip-label-font-size: 30px;
+
+// WindDirectionControl
+// ---------------------------------------
+@agate-winddirectioncontrol-icon-font-size: 210px;
+@agate-winddirectioncontrol-icon-height: 210px;
+@agate-winddirectioncontrol-icon-line-height: 180px;
```

Comment:
Are there any reason to use 180px instead of 210px?

Reply:
No, I don't think there is. I changed it to 210px.

Reply:
I looks that the icon positions in the center of the circle now. Thanks.